src/simcore_service_deployment_agent/git_url_watcher.py

```diff
@@ -277,12 +279,46 @@ async def _init_repositories(repos: list[GitRepo]) -> dict:
     return description
 
 
+async def _check_if_tag_on_branch(repo_path: str, branch: str, tag: str) -> bool:
+    cmd = [
+        "git",
+        "log",
+        "--tags",
+        "--simplify-by-decoration",
+        '--pretty="format:%ai %d"',
+    ]
+    try:
+        data = await run_cmd_line(cmd, repo_path)
+    except CmdLineError as e:
+        raise RuntimeError(
+            " ".join(cmd), "The command was invalid and the cmd call failed."
```

Comment:
Question: is this displayed as a unique string at runtime? Not sure how it deals with the comma

Reply:
this seems to work, comma concatenates